utils/demo/helpers/addManipulationBindings.ts

```diff
@@ -22,22 +24,35 @@ const { MouseBindings, KeyboardBindings } = csToolsEnums;
  * * Stack Scroll on Mouse Wheel, Primary+Alt
  * * Length Tool on fourth button
  */
-export default function addManipulationBindings(toolGroup, register = true) {
-  if (register) {
-    cornerstoneTools.addTool(LengthTool);
-    cornerstoneTools.addTool(StackScrollMouseWheelTool);
+export default function addManipulationBindings(
+  toolGroup,
+  options: {
```

Comment:
I think this is a good option as it allows us to add standard bindings for other types of views, like volume.